docker/api/network.py

```diff
@@ -33,6 +37,7 @@ def remove_network(self, net_id):
         url = self._url("/networks/{0}", net_id)
         res = self._delete(url)
         self._raise_for_status(res)
+        return True
```

Comment:
I wanted to make the `remove_network` function similar to the `remove_volume` function. Now, both return `True` if no exception is caused.

https://github.com/docker/docker-py/blob/master/docker/api/volume.py#L32-L36